### PR TITLE
upgrades and tower:reduce attack strength, armor.

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: factions/greeks/upgrades/pandoras_box/pandoras_box.xml [hunk 1]
```diff
@@ -20,9 +20,9 @@
 	<max-hp value="0"/>
 	<max-ep value="0"/>
 	<sight value="2"/>
-	<attack-strength value="75"/>
+	<attack-strength value="35"/>
```
Comment (andy5995) [inline factions/greeks/upgrades/pandoras_box/pandoras_box.xml:23]: This upgrade effects...

```
	<effects>
		<unit name="archer"/>
		<unit name="hoplite"/>
		<unit name="peltast"/>
		<unit name="cavalry"/>
		<unit name="dragon"/>
	</effects>
```

File: factions/greeks/upgrades/weapons/weapons.xml [hunk 2]
```diff
@@ -21,7 +21,7 @@
 	<max-hp value="0"/>
 	<max-ep value="0"/>
 	<sight value="0"/>
-	<attack-strength value="50"/>
+	<attack-strength value="20"/>
```
Comment (andy5995) [inline factions/greeks/upgrades/weapons/weapons.xml:24]: This upgrade affects...

```
	<effects>
		<unit name="archer"/>
		<unit name="hoplite"/>
		<unit name="peltast"/>
		<unit name="cavalry"/>
		<unit name="psiloi"/>
	</effects>
```

File: factions/greeks/units/tower/tower.xml [hunk 3]
```diff
@@ -79,9 +79,9 @@
 				<sound-file path="$COMMONDATAPATH/sounds/defense_tower_attack1.wav"/>
 				<sound-file path="$COMMONDATAPATH/sounds/defense_tower_attack2.wav"/>
 			</sound>
-			<attack-strength value="80"/>
-			<attack-var value="40"/>
-			<attack-range value="15"/>
+			<attack-strength value="60"/>
```
Comment (andy5995) [inline factions/greeks/units/tower/tower.xml:82]: I've reduced these because the units mentioned below will be weaker as well.

**Comment by titiger** [comment]
why do you want them weaker ?

**Comment by andy5995** [comment]
> why do you want them weaker ?

I think this would help balance it more, for a time in the future if we want to focus on balancing with other megapack factions. What do you think?

**Pull request merged**